### ISSUE TITLE
Reconsider accessibilty strategy for frontend output

### ISSUE THREAD

**Issue by mattcowan** [mattcowan/ligatures-and-typographic-styles-for-wordpress#20]
text that's used for screen readers needs to be at the same level as the opentype stylist block, additionally consider whether the accessibility fix we're making is the right one or if we should be using aria-content instead of a separate tag for the screen reader text.
